plugins/librato/measurement.go

```diff
@@ -38,6 +38,10 @@ func newMeasurement() *measurement {
 }
 
 func (m *measurement) floorTime(interval int64) {
+	if interval <= 0 {
+		log.Errorf("Attempted to floor measurement time with interval %d", interval)
+		return
```

Comment:
I think i was erring (pun not intended.... or is it???) on the side of getting measures to the remote service if at all possible.  In this case, it's not a huge deal to not floor the time, as its only purpose is really to align timestamps in librato as much possible.  So a log.Errorf seemed reasonable tiven that.